Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/security/KeycloakLoginUrlAuthenticationEntryPoint.java

```diff
@@ -67,18 +66,18 @@ public void setKeycloakLoginFormUrl(final String keycloakLoginFormUrl) {
         if ((this.keycloakLoginFormUrl == null) ||
                 (this.keycloakLoginFormUrl.contains(KEYCLOAK_REALM_PLACEHOLDER))) {
             final String keycloakRealm = System.getenv(ENV_KEYCLOAK_REALM_NAME_KEY);
```

Comment:
The configuration which is currently in environment variables should be moved to site.properties like everything else in the system.  We're already storing the database passwords there, so it's okay (or at least consistent) to store the Keycloak credentials there as well.

Reply:
As discussed, we'll do this sometime near future as Bassi is head under with stuff atm, and we want to get this across the line.